cypress/e2e/moderationDashboard.cy.ts

```diff
@@ -121,48 +125,62 @@ describe('Moderation Facility Submission Form', () => {
         })
 
         it('should be display error messages', () => {
-
-            cy.get('[data-testid="submission-form-nameEn"]').find('input').type('立川中央病院').tab()
+            //there is no cy.tab(), so manually need to shift focus from one input to another
```

Comment:
Let's remove this comment. Also please see the screen recording I shared of `.tab()` working with the plugin installed.

Seeing as this is passing on CI and only failing on your local machine, I think we should try to figure out why before making changes to these tests.